Resources/Prototypes/Catalog/Research/technologies.yml

```diff
@@ -213,27 +213,217 @@
-#   id: DirectEnergyTechnology
-#   description: Basically laser guns.
-#   icon:
-#     sprite: Objects/Weapons/Guns/Battery/taser.rsi
-#     state: icon
-#   requiredPoints: 15000
-#   requiredTechnologies:
-#     - SecurityTechnology
+- type: technology
+  name: technologies-security-basic
+  id: BasicSecurityTechnology
+  description: technologies-security-basic-description
+  icon:
+    sprite: Objects/Misc/handcuffs.rsi
+    state: handcuffs
+  requiredPoints: 5000
+  requiredTechnologies:
+    - BasicResearch
+  unlockedRecipes:
+    - Handcuffs
+    - Zipties
+    - RiotShield
+    - RiotLaserShield
+    - RiotBulletShield
+    - ClothingGlassesEyesSecurity
+    - FlashlightSeclite
+
+- type: technology
+  name: technologies-forensics-equipment
+  id: ForensicsEquipmentTechnology
+  description: technologies-forensics-equipment-description
+  icon:
+    sprite: Objects/Devices/forensic_scanner.rsi
+    state: forensicnew
+  requiredPoints: 5000
+  requiredTechnologies:
+    - BasicSecurityTechnology
+  unlockedRecipes:
+    - ForensicScanner
+    - ForensicPad
+
+- type: technology
+  name: technologies-speed-loader
+  id: SpeedLoaderTechnology
+  description: technologies-speed-loader-description
+  icon:
+    sprite: Objects/Weapons/Guns/Ammunition/SpeedLoaders/Magnum/magnum_speed_loader.rsi
+    state: base-6
+  requiredPoints: 5000
+  requiredTechnologies:
+    - ForensicsEquipmentTechnology
+    - BallisticMagazinesTechnology
+  unlockedRecipes:
+    - SpeedLoaderRubber
+    - SpeedLoaderMagnum
+    - SpeedLoaderHighVelocity
+
+- type: technology
+  name: technologies-ballistic-magazines
+  id: BallisticMagazinesTechnology
+  description: technologies-ballistic-magazines-description
+  icon:
+    sprite: Objects/Weapons/Guns/Ammunition/Magazine/Pistol/pistol_mag.rsi
+    state: icon
+  requiredPoints: 10000
+  requiredTechnologies:
+    - BasicSecurityTechnology
+  unlockedRecipes:
+    - MagazineRifle
+    - MagazinePistolSubMachineGunTopMounted
+    - MagazinePistol
+
+- type: technology
+  name: technologies-high-velocity-magazines
+  id: HighVelocityMagazinesTechnology
+  description: technologies-high-velocity-magazines-description
+  icon:
+    sprite: Objects/Weapons/Guns/Ammunition/Magazine/Pistol/pistol_mag.rsi
+    state: red-icon
+  requiredPoints: 10000
+  requiredTechnologies:
+    - BallisticMagazinesTechnology
+  unlockedRecipes:
+    - MagazineRifleHighVelocity
+    - MagazinePistolHighVelocity
+
+- type: technology
+  name: technologies-non-lethal
+  id: NonLethalTechnology
+  description: technologies-non-lethal-description
+  icon:
+    sprite: Objects/Weapons/Guns/Battery/disabler.rsi
+    state: base
+  requiredPoints: 5000
+  requiredTechnologies:
+    - BasicSecurityTechnology
+  unlockedRecipes:
+    - Disabler
+    - Flashbang
+    - Flash
+    - FlashPayload
+    - StunBaton
+    - Bola
+    - MagazineRifleRubber
+    - MagazinePistolRubber
+    - ShotgunShellBeanbag
+    - ShellTranquilizer
```

Comment:
consider splitting into "restraint tech" with the flashes, stun baton, disabler, bola, and stun bola, and "nonlethal ammunition" with all the rubber bullets and tranq shells.